lib/modules/manager/pip-compile/extract.spec.ts

```diff
@@ -0,0 +1,122 @@
+import { Fixtures } from '../../../../test/fixtures';
+import { fs } from '../../../../test/util';
+import { extractAllPackageFiles, extractPackageFile } from './extract';
+
+jest.mock('../../../util/fs');
+
+function getSimpleRequirementsFile(command: string, deps: string[] = []) {
+  return `#
+# This file is autogenerated by pip-compile with Python 3.11
+# by the following command:
+#
+#    ${command}
+#
+
+${deps.join('\n')}`;
+}
+
+describe('modules/manager/pip-compile/extract', () => {
+  beforeEach(() => {});
+
+  describe('extractPackageFile()', () => {
+    it('returns object for requirements.in', () => {
+      expect(
+        extractPackageFile(
+          Fixtures.get('requirementsWithHashes.txt'),
+          'requirements.in',
+          {},
+        ),
+      ).toBeObject();
```

Comment:
can yu validate more explicit? so use `toEqual`?

Reply:
This indirectly tests if pip_requirements extract was invoked based on the name. I'm not interested in a specific valid object because that what other managers test. This should test if a proper manager was assigned based on a filename.

Reply:
we should always be explicit, I see no other validation of the results from that function. you can use toMatchObject to do partial matching. it should at least find all deps inside

Reply:
I will check for deps returned